.github/actions/paths-filter/action.yml

```diff
@@ -83,16 +84,19 @@ runs:
           - '**/src/test/**'
 
         maven-change:
-          - 'bom/*'
+          - 'bom/**'
           - 'build-tools/**'
           - 'dist/**'
-          - 'parent/*'
-          - 'pom.xml'
+          - 'parent/**'
+          - '**/pom.xml'
```

Comment:
I just wanted to raise that it before it was actually intended that it only affected the root pom, as the use was to run everything when parent or root changes. 

Not sure whether we still want this, but some thought. Now it will trigger even if a pom in Operate or Optimize deep down changes. 🤔

Reply:
I added this filter in https://github.com/camunda/camunda/commit/d4ba3e5adb5abc8651fb2ddb9316515833c8de78 and back then my (wrong) understanding was that it would actually trigger on any `pom.xml` in the subtree. The filter was intended to capture any Maven-related change.

This filter is currently used to trigger Java compile, unit and integration tests which could be affected by the root pom or subtree poms as well right (changed properties/goals)? From my non-Java-expert POV I think it makes sense to trigger on those but happy to be overruled by you as Java experts.

I just want to avoid having another loophole - rather have execute too often.